management/job.yaml

```diff
@@ -0,0 +1,100 @@
+openapi: 3.0.1
+info:
+  title: Adzerk Management API - Advertisers
+  description: Advertiser related Adzerk Management API
+  version: '1.0'
+servers:
+  - url: 'https://api.adzerk.net'
+tags:
+  - name: jobs
+    description: Create Jobs
+paths:
+  /v1/job/{jobId}:
```

Comment:
This is a get individual job endpoint.

Reply:
Was considering that my brain forgot porting from `job.yaml` to `openapi-3.yaml`. But this proves I duplicated and did not update `description` nor `operationId` for the single job getter.
Thank you for capturing both!